src/components/CallToContribute.tsx

```diff
@@ -87,37 +40,71 @@ const CallToContribute: React.FC<IProps> = ({ editPath }) => (
       ░▐▒▒▒▒▒▒▒▒▒▒▒▒▒▒░▒░▒░▒▒▄▒▒▐░░ ░░▀▄▒▒▒▒▒▒▒▒▒▒▒░▒░▒░▒▄▒▒▒▒▌░░
       ░░░░▀▄▒▒▒▒▒▒▒▒▒▒▄▄▄▀▒▒▒▒▄▀░░░ ░░░░░░▀▄▄▄▄▄▄▀▀▀▒▒▒▒▒▄▄▀░░░░░
       ░░░░░░░░░▒▒▒▒▒▒▒▒▒▒▀▀░░░░░░░░
-    </ImageColumn>
-    <Column>
-      <Title>
+    </Flex>
+    <Flex
+      direction="column"
+      flexGrow={1}
+      flexShrink={1}
+      flexBasis="50%"
+      p={4}
+      color="text"
+      textAlign={{ base: "center", lg: "left" }}
+    >
+      <Heading
+        as="h2"
+        fontFamily="monospace"
+        textTransform="uppercase"
+        bg="border"
+        p={1}
+        fontSize="2rem"
+        lineHeight={1.4}
+      >
         <Translation id="page-calltocontribute-title" />
-      </Title>
-      <Description>
+      </Heading>
+      <Text lineHeight="140%" color="text" fontFamily="monospace">
         <Translation id="page-calltocontribute-desc-1" />
-      </Description>
-      <Description>
+      </Text>
+      <Text lineHeight="140%" color="text" fontFamily="monospace">
         <Translation id="page-calltocontribute-desc-2" />
-      </Description>
-      <Description>
+      </Text>
+      <Text lineHeight="140%" color="text" fontFamily="monospace">
         <Translation id="page-calltocontribute-desc-3" />{" "}
         <Link to="https://www.notion.so/efdn/Writer-template-4b40d196cde7422ca6a2091de33550bd">
           <Translation id="page-calltocontribute-link" />
         </Link>
-      </Description>
-      <Description>
+      </Text>
+      <Text lineHeight="140%" color="text" fontFamily="monospace">
         <Translation id="page-calltocontribute-desc-4" />{" "}
         <Link to="https://discord.gg/CetY6Y4">
           <Translation id="page-calltocontribute-link-2" />
         </Link>{" "}
-      </Description>
-      <GithubButton to={editPath}>
-        <GithubIcon name="github" />{" "}
-        <span>
-          <Translation id="page-calltocontribute-span" />
-        </span>
-      </GithubButton>
-    </Column>
-  </StyledCard>
+      </Text>
+      <ButtonLink
+        as={ButtonLink}
```

Comment:
If I remove this line the background style is ignored :confused:

Reply:
Yes, lets remove all those custom style props we have in this particular button.

With `<ButtonLink to={editPath} leftIcon={<Icon as={FaGithub} />}>` we should have almost the same styles.

Reply:
I think it is a good opportunity to clean this button and leave the standard solid styled button we are using from now on.

Reply:
Ok let's do that !